functionalTest/test/pageobjects/dashboard.page.js

```diff
@@ -0,0 +1,34 @@
+const Page = require('./page');
```

Comment:
Curious why this code is using require instead of ES6 import. If there is no particular need, it would be good to keep it consistent and use imports https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import

Reply:
implemented ES6 import in the latest changes.